E721 for instance checks use `isinstance()` of flake8 occurred.

**Describe the bug**
githubへのpush時にflake8のE721 for instance checks use `isinstance()`が発生しました。

**To Reproduce**
現在のmain(7/24)(0860a9a)を再pushしても起こると思います。
ドキュメントだけを変更したfeature/experimental/mpi(7/31)(31cb4c0)のpush時に発生しました。

flake8のversionが6.0.0から6.1.0にあがったことが原因と推測しています。

ok main(7/24)(0860a9a)のpush時のactionsのメッセージの抜粋
```
Successfully installed black-23.7.0 click-8.1.6 flake8-6.0.0 isort-5.12.0 mccabe-0.7.0 mypy-extensions-1.0.0 packaging-23.1 pathspec-0.11.1 platformdirs-3.9.1 pycodestyle-2.10.0 pyflakes-3.0.1 tomli-2.0.1 typing-extensions-4.7.1
```

error feature/experimental/mpi(7/31)(31cb4c0)のpush時のactionsのメッセージの抜粋
```
Successfully installed black-23.7.0 click-8.1.6 flake8-6.1.0 isort-5.12.0 mccabe-0.7.0 mypy-extensions-1.0.0 packaging-23.1 pathspec-0.11.2 platformdirs-3.10.0 pycodestyle-2.11.0 pyflakes-3.1.0 tomli-2.0.1 typing-extensions-4.7.1

aiaccel/abci/qsub.py:55:8: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
aiaccel/abci/qsub.py:58:10: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
aiaccel/util/easy_visualizer.py:109:16: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
aiaccel/util/easy_visualizer.py:191:16: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
```

**Expected behavior**
指摘された部分をisinstance()に修正するのが良いと思います。

--- a/aiaccel/abci/qsub.py
+++ b/aiaccel/abci/qsub.py
@@ -52,10 +52,10 @@ def create_qsub_command(config: DictConfig, runner_file: Path) -> list[str]:
         return command
 
     # add option
-    if type(job_execution_options) == str:
+    if isinstance(job_execution_options, str):
         for cmd in job_execution_options.split(" "):
             command_tmp.insert(-1, cmd)
-    elif type(job_execution_options) == list or type(job_execution_options) == ListConfig:
+    elif isinstance(job_execution_options, list) or isinstance(job_execution_options, ListConfig):
         for option in job_execution_options:
             for cmd in option.split(" "):
                 command_tmp.insert(-1, cmd)

--- a/aiaccel/util/easy_visualizer.py
+++ b/aiaccel/util/easy_visualizer.py
@@ -106,7 +106,7 @@ class EasyVisualizer:
         Raises:
             Exception: An out-of-target color is specified.
         """
-        if not type(colors) == list:
+        if not isinstance(colors, list):
             raise Exception
 
         self.plot_config["colors"] = []
@@ -188,7 +188,7 @@ class EasyVisualizer:
         Returns:
             Sorted list, or None.
         """
-        if not type(data) == list:
+        if not isinstance(data, list):
             return None
 
         best_values = []

